openpathsampling/netcdfplus/cache.py

```diff
@@ -374,7 +374,7 @@ class LRUChunkLoadingCache(Cache):
 
     """
 
-    def __init__(self, chunksize=100, max_chunks=100, variable=None):
+    def __init__(self, chunksize=256, max_chunks=8192, variable=None):
```

Comment:
Just curious: how much optimization did you do with these chunking numbers? on what kind of system? It might be really nice to have some benchmarks of this.